src/admin-views/tribe-options-general.php

```diff
@@ -160,6 +160,40 @@
 			'type'        => 'html',
 			'html'        => '<fieldset class="tribe-field tribe-field-html"><legend>' . esc_html__( 'Duplicate Venues &amp; Organizers', 'the-events-calendar' ) . '</legend><div class="tribe-field-wrap">' . Tribe__Events__Amalgamator::migration_button( esc_html__( 'Merge Duplicates', 'the-events-calendar' ) ) . '<p class="tribe-field-indent description">' . esc_html__( 'You might find duplicate venues and organizers when updating The Events Calendar from a pre-3.0 version. Click this button to automatically merge identical venues and organizers.', 'the-events-calendar' ) . '</p></div></fieldset><div class="clear"></div>',
 		),
+		'trashPastEvents'  => array(
+			'type'            => 'dropdown',
+			'label'           => esc_html__( 'Move to trash events older than', 'the-events-calendar' ),
+			'tooltip'         => esc_html__( 'This option allows you to automatically move past events to trash.', 'the-events-calendar' ),
+			'validation_type' => 'options',
+			'size'            => 'small',
+			'default'         => null,
+			'options'         => array(
+				null => esc_html__( 'Disabled', 'the-events-calendar' ),
+				3    => esc_html__( '3 months', 'the-events-calendar' ),
+				6    => esc_html__( '6 months', 'the-events-calendar' ),
+				9    => esc_html__( '9 months', 'the-events-calendar' ),
+				12   => esc_html__( '1 year', 'the-events-calendar' ),
+				24   => esc_html__( '2 years', 'the-events-calendar' ),
+				36   => esc_html__( '3 years', 'the-events-calendar' ),
+			),
+		),
+		'deletePastEvents' => array(
```

Comment:
If we can avoid creating a new option with CamelCase we should. Also we should store this key on a class for better re usability.